src/hooks/useRoomRecentUsersList.ts

```diff
@@ -33,5 +44,8 @@ export const useRoomRecentUsersList = ({
     locationNames: roomSlugs,
   });
 
-  return recentLocationUsers;
+  const filteredRecentUsers = recentLocationUsers?.filter(
+    (userLocation) => !!userLocation.length
+  );
+  return filteredRecentUsers;
```

Comment:
I don't think we need this check here given the way the hook works:

![image](https://user-images.githubusercontent.com/63194656/131360906-34b7557a-ed05-459e-8033-a8a43d150216.png)